js/modal.js

```diff
@@ -1,38 +1,76 @@
-function openEventCreationModal() {
-    const modal = document.getElementById("eventModal");
+import { saveEventToLocalStorage, loadEventsForCurrentWeek } from './events.js';
+import { getFirstDayOfTheWeek } from './utils.js';
+export const initModal = () => {
+  const modal = document.getElementById("eventModal");
+  const eventDetailsModal = document.getElementById("eventDetailsModal");
+
+  const openEventCreationModal = () => {
     modal.style.display = "block";
-}
-function closeEventCreationModal() {
-    const modal = document.getElementById("eventModal");
+  };
+
+  const closeEventCreationModal = () => {
     modal.style.display = "none";
-}
-function validateEventForm(event) {
+  };
+
+  const openModal = (date) => {
+    //kai reikes jeigu reikes
```

Comment:
: )
I think we already need it.
I am not sure if you need both openEventCreationModal and openModal though.